docs/shared-content/installation/server-downloads.include.md

```diff
@@ -1,15 +1,8 @@
-### Download the long-term support Octopus Server {#slow-lane}
+### Download the Octopus Server
 
-The latest long-term support version of Octopus Deploy:
+The latest version of the Octopus Deploy Server:
 
-- [Octopus Server x64](https://octopus.com/downloads/slowlane/WindowsX64/OctopusServer)
-- [Octopus Server 32-bit/x86](https://octopus.com/downloads/slowlane/WindowsX86/OctopusServer)
+- [Octopus Server](https://octopus.com/downloads/slowlane/WindowsX64/OctopusServer)
```

Comment:
I think this should go to https://octopus.com/downloads

Reply:
Done. Added the link to the text above.